Sources/UIWindow.swift

```diff
@@ -37,8 +37,8 @@ public class UIWindow: UIView {
             currentTouch.view = hitView
             currentTouch.gestureRecognizers = hitView.getRecognizerHierachy()
 
-            currentTouch.runTouchActionOnRecognizerHierachy { $0.touchesBegan(allTouches, with: event) }
             hitView.touchesBegan(allTouches, with: event)
+            currentTouch.runTouchActionOnRecognizerHierachy { $0.touchesBegan(allTouches, with: event) }
```

Comment:
I'm pretty sure this change is not correct. From my understanding, gesture recognizers need to be able to handle / cancel the touch _before_ touchesBegan gets called

Reply:
I was able to revert the change without breaking the UX, so it's fine for me to leave it as it was before